job_scripts/hpss/olcf_kronos_once.submit

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#SBATCH -A ast106
+#SBATCH -t 02:00:00
+#SBATCH --cluster dtn
+#SBATCH -N 1
+
+# do our archiving
+cd "$SLURM_SUBMIT_DIR" || exit
+
+# use srun so any control signals get sent to the child too
+srun ./kronos_process.sh once
```

Comment:
what does "once" do here?

Reply:
It archives the current files then exits, instead of waiting for new files to show up.

Reply:
ah I missed the comments in the code